src/Shared/UnitTests/FileUtilities_Tests.cs

```diff
@@ -392,6 +409,12 @@ public void NormalizePathEmpty()
         [Fact]
         public void NormalizePathBadUNC1()
         {
+            // UNC is not useful outside Windows
+            if (!NativeMethodsShared.IsWindows)
```

Comment:
Yeah, I think it's time we get that up and running.  Looks like there's some magic at https://github.com/dotnet/buildtools/tree/master/src/xunit.netcore.extensions to help us out.

(I'm willing to let this go in and clean up, personally.)